examples/react-native-expo-router-with-shared-routes/src/app/(tabs)/(home, feed, example)/post.tsx

```diff
@@ -0,0 +1,53 @@
+import Button from '@/components/Button';
+import { Link, useLocalSearchParams, useRouter, useSegments } from 'expo-router';
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+export default function Post() {
+  const segment = useSegments();
+  const router = useRouter();
+  const params = useLocalSearchParams<{ randomId: string }>();
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.main}>
+        <Text style={styles.title}>Post :)</Text>
+        <Text style={styles.subtitle}>This is page render inside the stack of the tab</Text>
+      </View>
+
+      <Button
+        onClick={() => {
+          router.push({
+            pathname: `/${segment[0]}/${segment[1]}/post`,
```

Comment:
could we named theses segments? for example use destructoring in `useSegments()`:

```jsx
const [tabs, segment] = useSegments()
```

Reply:
Hi, @rpmolina .
I changed all the useSegments around the app, i used the destructuring option that you suggest.
Check if its okay!